CDS/config/wlp/cds/server.xml

```diff
@@ -17,6 +17,7 @@
     <httpSession invalidationTimeout="8h" idReuse="true"/>
 
     <keyStore id="defaultKeyStore" password="{xor}FhwPHAswMDMs" />
+    <!--<ssl id="defaultSSLConfig" keyStoreRef="defaultKeyStore" trustDefaultCerts="true" />-->
```

Comment:
What does this do and why can't we enable it by default?

Reply:
It makes that the JVM/CDS server trusts the certificate of sentry.io. Normally we don't have any certificates in the store so there was an SSL error. If people don't use sentry I don't see a reason to insert that many certificates at once. I think it's fine to default change this (as the machine already trusts those certs) but wanted to limit the amount of new features as much as possible.